README.md

```diff
@@ -12,342 +14,117 @@
-    assetLoader: YamlAssetLoader(),
-    ```
-
-    ```dart
-    path: 'resources/langs/langs.yaml',
-    assetLoader: YamlSingleAssetLoader(),
-    ```
-
-  - Added support XML files.
-
-    ```dart
-    path: 'resources/langs',
-    assetLoader: XmlAssetLoader(),
-    ```
-
-    ```dart
-    path: 'resources/langs/langs.xml',
-    assetLoader: XmlSingleAssetLoader(),
-    ```
-
-- Added Code generation of localization files.
-
-  ```cmd
-  $ flutter pub run easy_localization:generate -h
-  -s, --source-dir     Source folder contains all string json files
-                      (defaults to "resources/langs")
-  -O, --output-dir     Output folder stores generated file
-                      (defaults to "lib/generated")
-  -o, --output-file    Output file name
-                      (defaults to "codegen_loader.g.dart")
-  -f, --format         Support json, dart formats
-                      [json (default), keys]
-  ```
-
-  - generate the json string static keys in a dart class
-
-    ```json
-    {
-      "msg_named": "{} مكتوبة باللغة {lang}",
-    }
-    ```
-
-    ```cmd
-    flutter pub run easy_localization:generate  -f keys -o locale_keys.g.dart
-    ```
-
-    ```dart
-    abstract class  LocaleKeys {
-      static const msg_named = 'msg_named';
-    }
-    ```
-
-    ```dart
-    Text(LocaleKeys.msg_named).tr(namedArgs: {'lang': 'Dart'}, args: ['Easy localization']),
-    ```
-
-  - generate the json Loader in a dart class
-  
-    ```cmd
-    flutter pub run easy_localization:generate
-    ```
-
-- fixed many issues.
-- Added named arguments.
-
-  ```json
-  "msg_named": "{} are written in the {lang} language",
-  ```
-
-  ```dart
-  Text(LocaleKeys.msg_named).tr(namedArgs: {'lang': 'Dart'}, args: ['Easy localization']),
-  ```
-
-### [2.1.0]
+## Features
 
-- Added Error widget.
-- fixed many issues.
-- Adopted Bloc architecture.
+- Easy translations for many languages
+- Load translations as JSON, CSV, Yaml, Xml using [easy\_localization\_loader](https://github.com/aissat/easy_localization_loader)
+- React and persist to locale changes
+- Supports `plural`, `gender`, nesting, RTL locales and more
+- Optional code generation
+- Error widget for missing translations
+- Extension methods on `Text`
+- Uses BLoC pattern
 
-### [2.0.2]
+See [example](example) and [CHANGELOG.md](CHANGELOG.md).
 
-- fixed many issues.
-- optimized and clean code for more stability.
+## Getting started
 
-### [2.0.1]
-
-- Added change locale dynamically `saveLocale` default value `true`.
-- fixed many issues.
-
-## Getting Started
-
-### Configuration
-
-Add this to your package's pubspec.yaml file:
+Add to your `pubspec.yaml`:
 
 ```yaml
 dependencies:
-  # stable version install from https://pub.dev/packages
-  easy_localization: <last_version>
-
-  # Dev version install from git REPO
   easy_localization:
-    git: https://github.com/aissat/easy_localization.git
+  easy_localization_loader:
 
 ```
 
-#### Load translations from local assets
```

Comment:
this part is important

Reply:
ive added it back with some fixes and force pushed it